default/python/turn_events/turn_events.py

```diff
@@ -6,6 +6,66 @@
 from universe_tables import MONSTER_FREQUENCY
 
 
+
+# tuples of galaxy setup options to pick from randomly
+low_to_high =  (fo.galaxySetupOption.low, fo.galaxySetupOption.medium, fo.galaxySetupOption.high)
+none_to_high = (fo.galaxySetupOption.low, fo.galaxySetupOption.medium, fo.galaxySetupOption.high, fo.galaxySetupOption.none)
```

Comment:
It was suggestion to rename variable.
Previous variable named form first element of tuple to last, this named form last to last but one. I see inconsistency here.